pubspec.yaml

```diff
@@ -32,6 +32,7 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   intl: ^0.17.0
+  provider: ^6.0.2
```

Comment:
I'm sure it will be useful but not used anywhere in this PR? :)

Reply:
Indeed. Let's take it out for now.